core/src/io/anuke/mindustry/ui/Links.java

```diff
@@ -27,6 +34,19 @@ private static void createLinks(){
         if(links == null){
             createLinks();
         }
+        if(!missingLocales.isEmpty()){
+            String key = "missinglocale";
+            if(missingLocales.size()==1){
+                key = "missinglocale.single";
+            }
+            String m = Core.bundle.getOrNull(key);
+            if(m==null)m="Locale key(s) missing:\n";
+            for(String s : missingLocales){
+                m = m+s;
+            }
+            String finalM = m;
+            Time.run(1f, ()->{ui.showErrorMessage(finalM);});
```

Comment:
`finalM` is used for nothing. `m` variable can be used for `Time.run`.
```java
            Time.run(1f, ()->{ui.showErrorMessage(m);});
```
*With my suggestion:*
```java
            Time.run(1f, () -> { ui.showErrorMessage(m); });
```

Reply:
m cannot be used in this case due to it being used in a lambda expression. (These need to be final or effectively final. I am unsure how else to achieve this.)